src/node_crypto.h

```diff
@@ -488,22 +488,21 @@ class Hmac : public BaseObject {
 
   static void Initialize(Environment* env, v8::Local<v8::Object> target);
 
- protected:
```

Comment:
I'm not sure about this. Not only would it create an inconsistency with the other crypto C++ APIs (since we'd now be exposing the 3 below methods to everyone), but I *think* doing so could even constitute a `semver-major` change?